karaf/features/src/main/feature/feature.xml

```diff
@@ -256,7 +256,7 @@
             https://github.com/square/okhttp/pull/2246
         -->
         <!-- TODO use ServiceMix bundle for okhttp when dependency on okio is fixed there -->
-        <bundle dependency="true">wrap:mvn:com.squareup.okhttp/okhttp/2.2.0$Import-Package=org.apache.http.*;resolution:=optional,android.util.*;resolution:=optional,okio;version="[1.2.0, 1.3.0)",*</bundle>
+        <bundle dependency="true">wrap:mvn:com.squareup.okhttp/okhttp/2.2.0$Import-Package=org.apache.http.*;resolution:=optional,android.util.*;resolution:=optional,okio;version=%27[1.2.0,1.3.0)%27,*</bundle>
```

Comment:
Tested with encoded double quotes (`%22`) but the feature verification was still failing. Not sure why unfortunately